client/src/components/DisplayTable.js

```diff
@@ -0,0 +1,141 @@
+import './DisplayTable.scss';
+import classNames from 'classnames';
+import { util_components } from '../panels/shared.js';
+const {
+  SortDirections,
+} = util_components;
+
+import { TM } from './TextMaker.js';
+
+export class DisplayTable extends React.Component {
+  constructor(props){
+    super();
+    this.state = {
+      sort_by: "label",
+      descending: true,
+      filter_by: undefined,
+      filter: undefined,
+      show_all: false,
+    };
+  }
+
+  header_click(col_name){
+    this.setState({
+      sort_by: col_name,
+      descending: (
+        this.state.sort_by === col_name ?
+          !this.state.descending :
+          true
+      ),
+    });
+  }
+
+  render(){
+    const {
+      column_keys,
+      data,
+      table_name,
+      label_col_header,
+      sort_keys,
+      table_data_headers,
+    } = this.props;
+    const {
+      sort_by,
+      descending,
+    } = this.state;
+  
+
+    // TODO: implement filtering
+    const sorted_filtered_data = _.chain(data)
```

Comment:
Nope, this table doesn't allow for filtering. I should file this as an enhancement issue but I wanted to keep the comment because right now the variable name lies.

Reply:
I guess it would be better to just rename the variable and remove the comment....fine I will do that.